src/modules/integrator-guide/pages/automation/workflows/examples/messagebus-read-write.adoc

```diff
@@ -0,0 +1,77 @@
+= Messagebus Queue
+:attachment-path: ../../../_attachments/automation/workflows/
+:page-noindex: true
+
+Messagebus queue mechanism complements the workflows in a way that any asynchronous computation can be offloaded to another workflow or workflow-extendable process. +
+This example will be using the xref:automation/workflows/index.adoc[], Integration Gateway xref:api-gw/profiler.adoc[] and the xref:developer-guide:corteza-server/messagebus.adoc[] subsystem.
+
+.In this example we will:
+
+ - create a queue
+ - create an endpoint
+ - write to queue via workflow
+ - read from queue via workflow
+    * send a payload to an endpoint
+ - view endpoint payload in profiler
```

Comment:
Use `*` for lists. To indent an item, use`**`.
Remove the empty line below `.In this example we will:`